bite-size-notes/week-6-e-Perception_Aware_MPC_Paper_review.md

```diff
@@ -0,0 +1,107 @@
+Later, these equations can be used to express the projection velocity as a function of the quadrotor state and input vectors as we are interested in reducing the velocity of its projection onto the image plane. 
+
+## Problem Formulation
+
+The unified optimization problem can be formulated as following:
+
+$$
+\begin{align*}
+& \min _{\mathbf{u}} \int_{t_{0}}^{t_{f}} \mathcal{L}_{a}(\mathbf{x}, \mathbf{u})+\mathcal{L}_{p}(\mathbf{z}) d t  \tag{1}\\
+& \text { subject to } \quad \mathbf{r}(\mathbf{x}, \mathbf{u}, \mathbf{z})=0 \\
+& \mathbf{h}(\mathbf{x}, \mathbf{u}, \mathbf{z}) \leq 0
+\end{align*}
+$$
+
+Where
+
+$$
+\begin{aligned}
+& \dot{\mathbf{x}}=\boldsymbol{f}(\mathbf{x}, \mathbf{u}) \text { is the system dynamics } \\
+& \mathbf{z}=\boldsymbol{f}_{p}(\mathbf{x}, \mathbf{u}, \sigma) \cdot \text { is the state vector for the perception dynamics } \\
+& \mathcal{L}_{a}(\mathbf{x}, \mathbf{u}) \text { defines an action cost } \\
+& \mathcal{L}_{p}(\mathbf{z}) \text { defines a cost for perception objectives. }
+\end{aligned}
+$$
+
+After deriving the quadrotor dynamics and perception objectives the following cost function was employed in the Model Predictive Control :
+
+$$
+\begin{align*}
+\mathcal{L}= & \overline{\mathbf{x}}_{N}^{\top} \mathcal{Q}_{x, N} \overline{\mathbf{x}}_{N}+ \\
+& +\sum_{i=1}^{N-1}\left(\overline{\mathbf{x}}_{i}^{\top} \mathcal{Q}_{x, i} \overline{\mathbf{x}}_{i}+\overline{\mathbf{z}}_{i}^{\top} \mathcal{Q}_{p, i} \overline{\mathbf{z}}_{i}+\overline{\mathbf{u}}_{i}^{\top} \mathcal{R}_{i} \overline{\mathbf{u}}_{i}\right) \tag{10}
+\end{align*}
+$$
+
+where the values  $\overline{\mathbf{x}}, \overline{\mathbf{z}}, \overline{\mathbf{u}}$ refer to the difference with respect to the reference of each value.
+
+The inputs $\mathbf{u}$, consisting of $c$ and $\boldsymbol{\Omega}_{B}$, as well as the velocity $\mathbf{v}_{W B}$ are limited by the constraints:
+
+$$
+\begin{gather*}
+c_{\min } \leq c \leq c_{\max }  \tag{11}\\
+-\Omega_{\max } \leq \boldsymbol{\Omega}_{B} \leq \Omega_{\max }  \tag{12}\\
+-v_{\max } \leq \mathbf{v}_{W B} \leq v_{\max } \tag{13}
+\end{gather*}
+$$
+
+where $c_{\min }, c_{\max }, \Omega_{\max }, v_{\max } \in \mathcal{R}_{+} \cdot$
+
+## Conclusion
+
+The aforementioned cost function and constraints used in PAMPC are then used with the hardware of the quadrotors to execute the control action in real time. The authors proved successful results using this approach via experimentation. An example of it was a room with
+obstacles (boxes) on the ground where the quadrotors were able to manage to encircle the obstacles and control the speed when they were around.
+
+![](https://cdn.mathpix.com/cropped/2024_05_09_879380217ac9dd082806g-3.jpg?height=835&width=1645&top_left_y=396&top_left_x=240)
+
+The figure above shows the quadrotors maintaining their velocity at the same time maximizing their vision space around objects.
+
+## References
```

Comment:
The reference is clear since it mentioned the paper the writer used along with the link